Text-Based-Adventure/Text-Based-Adventure/main.swift

```diff
@@ -0,0 +1,459 @@
+//
+//  main.swift
+//  Text-Based-Adventure
+//
+//  Created by Kelby Mittan on 9/30/19.
+//  Copyright © 2019 Kelby Mittan. All rights reserved.
+//
+
+import Foundation
+
+
+print("Enter your name!!")
+let enterName = readLine()
+
+print()
+
+print("Hello \(enterName ?? "playa"), Welcome to Ascend NYC..... on a scale of \"1-10\" how adventurous are you feeling about treking to the top of a New York City landmark today?")
+
+let howAdventurous = readLine()
+
+let howAdenturousAsAnInt = Int(howAdventurous ?? "") ?? 7
```

Comment:
This is great naming of your variable, and it does exactly as you said, it casts anything inside to be an Int which means that the user can enter a String or Char and the outcome will always be 7, which kind of removes the user's ability to choose.  Just be mindful that if you give a choice to a user, they should be able to take it.